README.md

```diff
@@ -30,7 +28,7 @@ Asynchronous component to enable async Knative service calls.
     kubectl get kservice helloworld-sleep
     ```
     
-1. (Optional) If you wanted every service to respond to the `Prefer: respond-async` header asynchronously, you can configure Knative Serving to use the proper class for every service.
+1. (Optional) If you wanted every service created by knative to respond to the `Prefer: respond-async` header, you can configure Knative Serving to use the async ingress class for every service.
```

Comment:
Would it make sense to also mention that you can also undo by removing or changing the annotation? Also if they do so do they have to remove the Redis component or can they leave it optionally in case they want to re-enable later?

Reply:
The redis component wouldn't need to be removed for their Knative deploy to be "back to normal." I think it *does* make sense for us to add some uninstall instructions for the component though.  I'll open up an issue for that for now.

Reply:
https://github.com/knative-sandbox/async-component/issues/57

Reply:
updated to include undo instructions for this as well.